sound/soc/sof/intel/hda-stream.c

```diff
@@ -363,12 +363,30 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 			  sd_offset + SOF_HDA_ADSP_REG_CL_SD_CBL,
 			  hstream->bufsize);
 
+	/* couple host and link DMA */
+	snd_sof_dsp_update_bits(sdev, HDA_DSP_PP_BAR,
+				SOF_HDA_REG_PP_PPCTL,
+				mask, 0);
+
+	/* disable DSP features */
+	snd_sof_dsp_update_bits(sdev, HDA_DSP_PP_BAR, SOF_HDA_REG_PP_PPCTL,
+				SOF_HDA_PPCTL_GPROCEN, 0);
```

Comment:
Is this must according to your test? In Marcin's description, only PROCEN(for corresponding stream index) need to be set.

Reply:
each stream would call this function, right ?

Reply:
The issue could be fixed without this code

Reply:
right, each stream will call this.

Reply:
As @keyonjie mentioned it, it's very odd that we would use the general bit, that might impact already running streams. Please use PROCEN.

Reply:
yes, PROCEN is better and I will refine and test it.